arangod/Aql/DocumentProducingHelper.h

```diff
@@ -177,23 +178,22 @@ struct DocumentProducingFunctionContext {
   InputAqlItemRow const& _inputRow;
   OutputAqlItemRow* _outputRow;
   RegisterId const _outputRegister;
```

Comment:
RegisterId is an unsigned int (i.e., usually 32bit), so it should be moved before the bools.